app/services/email_sender/notify_sender.rb

```diff
@@ -0,0 +1,37 @@
+require "notifications/client"
+
+module EmailSender
+  class NotifySender
+    def call(options)
+      unpack(options)
+      raise ArgumentError, "address is required" if @address.nil?
+      send
+    end
+
+  private
+    attr_accessor :address, :template_id
+
+    def unpack(options)
+      raise ArgumentError, "call expects a hash" unless options.is_a?(Hash)
+      @address = options[:address]
+      @template_id = options[:template_id] || notify_config.fetch(:template_id)
```

Comment:
I would maybe move `notify_config.fetch(:template_id)` into its own method called `default_template_id`, for clarity.